### PR TITLE
Add contex info about concurrent LOAD CSV

### PR DESCRIPTION
**Pull request by antejavor**
### Description

Add context for concurrent Load CSV

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
https://github.com/memgraph/documentation/issues/494

Closes https://github.com/memgraph/documentation/issues/495

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/data-migration/csv.mdx [hunk 1]
```diff
@@ -180,8 +180,14 @@ You can switch between modes within the session using the following query:
 STORAGE MODE IN_MEMORY_{TRANSACTIONAL|ANALYTICAL};
 ```
 
-When in the analytical storage mode, **don't** import data using multiple
-threads.
+If you use `IN_MEMORY_ANALYTICAL` mode and have nodes and relationships stored in
+ separate CSV files, you can run multiple concurrent `LOAD CSV` queries to import data even faster.
+In order to achieve the best import performance, split your nodes and edge
```
Comment (katarinasupe) [inline pages/data-migration/csv.mdx:185]:
```suggestion
In order to achieve the best import performance, split your nodes and relationships
```

File: pages/data-migration/csv.mdx [hunk 2]
```diff
@@ -180,8 +180,14 @@ You can switch between modes within the session using the following query:
...
+files into smaller files and run multiple `LOAD CSV` queries in parallel. 
+The key is to run all `LOAD CSV` queries for nodes first. After that, run 
+all relationship `LOAD CSV` queries.
```
Comment (katarinasupe) [inline pages/data-migration/csv.mdx:188]:
```suggestion
The key is to run all `LOAD CSV` queries, which create nodes first. After that, run 
all `LOAD CSV` queries that create relationships. 
```

File: pages/data-migration/csv.mdx [hunk 3]
```diff
@@ -180,8 +180,14 @@ You can switch between modes within the session using the following query:
...
-When in the analytical storage mode, **don't** import data using multiple
-threads.
+If you use `IN_MEMORY_ANALYTICAL` mode and have nodes and relationships stored in
+ separate CSV files, you can run multiple concurrent `LOAD CSV` queries to import data even faster.
+In order to achieve the best import performance, split your nodes and edge
+files into smaller files and run multiple `LOAD CSV` queries in parallel. 
+The key is to run all `LOAD CSV` queries for nodes first. After that, run 
+all relationship `LOAD CSV` queries. 
+
+Read more about this in [import best pratices](TBD)
```
Comment (katarinasupe) [inline pages/data-migration/csv.mdx:190]:
```suggestion
Read more about this in the [import best practices](./best-practices) 
```

Comment (katarinasupe) [reply pages/data-migration/csv.mdx:190]: we'll update it later if needed

**Review by katarinasupe — CHANGES_REQUESTED** [review]

File: pages/data-migration/csv.mdx (hunk 3 above)
Comment (antejavor) [reply pages/data-migration/csv.mdx:190]: Let's updated the readme there and results, then we will just add links to places.

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**